src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.SslProvider.cs

```diff
@@ -99,6 +101,28 @@ private static void SetSslOptionsForSupportedBackend(EasyRequest easy, ClientCer
                 }
             }
 
+            private static void SetSslOptionsForCertificateStore(EasyRequest easy)
+            {
+                // Support specifying certificate directory/bundle via environment variables: SSL_CERT_DIR, SSL_CERT_FILE.
+                string sslCertDir = Environment.GetEnvironmentVariable("SSL_CERT_DIR");
+                if (sslCertDir != null)
+                {
+                    easy.SetCurlOption(Interop.Http.CURLoption.CURLOPT_CAPATH, sslCertDir);
```

Comment:
Are there any tests that we can add for this?

Reply:
A test could be to SSL_CERT_DIR to an empty folder and then `HttpGetAsync("https://www.microsoft.com")`. This will now throw since the certificate can no longer be validated.
Do you have a suggestion on how to deal with environment variables (global state) in the tests?
Perhaps we can put these tests in a separate project and ensure only one test runs at a time?
https://github.com/dotnet/corefx/issues/23150 will need something like this too for validation.

Reply:
> Do you have a suggestion on how to deal with environment variables (global state) in the tests?

The RemoteInvoke utility we use in tests runs the code in another process, and am overload lets you pass in Process starting, including the environment variables to set. You can see an example of that in a few other http tests (e.g. for respecting proxy env vars).

Reply:
@Priya91 I've added a test. Please take a look.